### PR TITLE
 [Improvement-11834] Upgrade docker base image to support python3.9

### PR DESCRIPTION
**Pull request by danielfree**
## Purpose of the pull request

Upgrade the docker base image to debian 11 bullseye, so when users want to install python3 via `apt udpate && apt install python3`, the installed version will be python3.9 instead of python3.7 as in debian 10 buster.

## Brief change log

change  `openjdk:8-jre-slim-buster` to `openjdk:8-jre-slim-bullseye`

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: dolphinscheduler-standalone-server/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-jre-slim-buster
+FROM openjdk:8-jre-slim-bullseye
```
Comment (SbloodyS) [inline dolphinscheduler-standalone-server/src/main/docker/Dockerfile:18]: Please upgrade all dockerfiles. Including other services.

Comment (danielfree) [reply dolphinscheduler-standalone-server/src/main/docker/Dockerfile:18]: sure, thx

**Review by SbloodyS — APPROVED** [review]
LGTM

**Comment by SbloodyS** [comment]
Thanks for your contribution. This is a good beginning. Considering that it's your first contribution, I think we can get deep communication, you can contact me through mail or add WeChat(SbloodyS), when mail or added, please tell me who you are. I think I can help to familiar with the Dolphinscheduler if you meet with problems.

**Pull request merged**